i18n/de_DE/messages/messages_laika.json

```diff
@@ -175,13 +175,13 @@
   "donation_form_section_continue": "Weiter",
   "donation_form_section_headline": "Spenden",
   "donation_form_section_overview_title": "Übersicht",
-  "donation_form_street_error": "Könnten Sie bitte Ihre Straße überprüfen? Da scheint es einen Fehler zu geben.",
+  "donation_form_street_error": "Bitte geben Sie eine Straße an.",
   "donation_form_street_label": "Straße und Hausnummer",
   "donation_form_street_number_warning": "Wir haben hier eine Hausnummer erwartet. Falls diese noch fehlt, ergänzen Sie bitte Ihre Eingabe.",
   "donation_form_street_placeholder": "z.B. Musterstraße 12",
   "donation_form_summary_external_payment": "Sie werden nach der Auswahl von „Jetzt Spenden” auf die Seite unseres Zahlungsdienstleisters weitergeleitet.",
   "donation_form_summary_bank_transfer_payment": "Nach der Auswahl von „Jetzt Spenden” erhalten Sie unsere Kontoverbindung sowie einen besonderen Verwendungszweck, die Sie für Ihre Überweisung benötigen.",
-  "donation_form_zip_error": "Wir haben hier eine 4 oder 5 stellige Postleitzahl erwartet. Bitte überprüfen Sie Ihre Eingabe.",
+  "donation_form_zip_error": "Bitte geben sie eine 4 oder 5-stellige Postleitzahl an.",
```

Comment:
"Bitte geben Sie eine ..." is correct. "Sie" instead of "sie"

Reply:
Sollte jetzt repariert sein